src/app/services/iron/iron-http-interceptor.ts

```diff
@@ -65,24 +64,40 @@ export class IronHttpInterceptor implements HttpInterceptor {
         );
     }
 
+    private isDecryptable(response: HttpResponse<any>): boolean {
```

Comment:
You should be able to write this as: 

```javascript
    private isDecryptable(response: HttpResponse<any>): boolean {
        return response.body instanceof Array ? response.body.every((item) => EncryptedDocument.is(item)) : EncryptedDocument.is(response.body);
    }
```

just to clean up the code a little bit.